dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/enrollment/AbstractEnrollmentService.java

```diff
@@ -934,6 +934,16 @@ private void updateCoordinates( Program program, Enrollment enrollment, ProgramI
         }
     }
 
+    private boolean doValidationOfMandatoryAttributes( User user )
+    {
+        if ( user != null && user.isAuthorized( Authorities.F_IGNORE_TRACKER_REQUIRED_VALUE_VALIDATION.getAuthority() ) )
```

Comment:
This will look better be better as a one-liner and negated / used the opposite way: 

private boolean skipValidation...(User)
{
  return user != null && user.isAuthorized( Authorities.F_IGNORE_TRACKER_REQUIRED_VALUE_VALIDATION.getAuthority() );
}